mycity/mycity/intents/farmers_market_intent.py

```diff
@@ -0,0 +1,56 @@
+"""
+Farmers Market Intent
+
+"""
+import mycity.utilities.gis_utils as gis_utils
+import logging
+import datetime
+from mycity.mycity_response_data_model import MyCityResponseDataModel
+from .custom_errors import BadAPIResponse
+
+logger = logging.getLogger(__name__)
+
+BASE_URL = 'https://services.arcgis.com/sFnw0xNflSi8J0uh/ArcGIS/rest/' \
+           'services/Farmers_Markets_and_Fresh_Trucks/FeatureServer/0'
+QUERY = {'where': '1=1', 'out_sr': '4326'}
+DAY = datetime.datetime.today().strftime('%A')
+
+
+def get_farmers_markets_today(mycity_request):
+    """
+    Get all available farmers markets today
+
+    :param mycity_request: MyCityRequestDataModel object
+    :return: MyCityResponseObject
+    """
+    mycity_response = MyCityResponseDataModel()
+
+    # List all available farmers markets today
+    markets = gis_utils.get_features_from_feature_server(BASE_URL, QUERY)
+
+    try:
+        # Loop through the list of available farmers markets at a certain day
+        markets_unique = []
+        for m in markets:
+            if m not in markets_unique and m['attributes']['Day_'] == DAY:
+                markets_unique.append(m)
+
+        response = 'Avaiable farmers markets today are:\n'
+        for m in markets_unique:
+            response += m['attributes']['Name'] + ' located at ' + \
+                        m['attributes']['Location'] + ' from ' + \
+                        m['attributes']['Start'] + '. '
```

Comment:
Does using a period vs. comma here have any effect on the way Alexa reads the result?

Reply:
Honestly, I did not find any difference when using `comma` or `period`. I was expecting that periods would be a slightly longer pause, but I cannot notice any difference (i used an Android application to record the sound dB and the valleys seem to be the same)